### DISCUSSION TITLE
Question about Chrome/Chromium manifest v3.

### DISCUSSION THREAD

**Discussion by smordarski** [uBlockOrigin/uBOL-home#88] [Q&A]
I saw that google posted [this](https://developer.chrome.com/blog/resuming-the-transition-to-mv3/) about resuming manifest v3. From this [post](https://developer.chrome.com/blog/chrome-120-beta-whats-new-for-extensions/#closing-the-platform-gap) made by Google, it says that the new userScripts API can:

> User scripts are (usually relatively small) snippets of code that extensions can inject into web pages in order to modify the page's appearance or behavior.

I am wondering if the new userScripts API will allow uBlock Origin with manifest v3 to load external filter lists such as my own. I know you have to enable developer mode but I wonder if this will help ad blockers such as uBlock Origin work on manifest v3.

**Comment by garry-ut99** [comment]
> to load external filter lists such as my own

Duplicate, see:
- https://github.com/uBlockOrigin/uBOL-home/discussions/153#discussioncomment-10105530
